openshift/templates/api/config/bc/invite-email.html

```diff
@@ -110,6 +89,13 @@
     above.
   </li>
 
+  <li>
+    SafeEntryBC - from your laptop or tablet click on the following link:
+    <a href="https://vonx.io/safeentry/"
+      >https://vonx.io/safeentry/</a
+    >.
+  </li>
```

Comment:
The link to SafeEntry is not really to "demo" service, more like a documentation page.

Would it be best to add it elsewhere rater than under the demo services list?